lib/install/management/appliance.go

```diff
@@ -465,8 +465,8 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 			Args: []string{
 				"/sbin/vicadmin",
 				"-docker-host=unix:///var/run/docker.sock",
-				// FIXME: hack during config migration
-				"-insecure",
+				fmt.Sprintf("-insecure=%t", conf.Insecure),
```

Comment:
Ah I didn't think of that, and passing the flags around was the most painful part of this whole thing ;)  I'll undo the flag passing.